### DISCUSSION TITLE
How could I use the inverse of the "The Tall Layout", therefore split left oriented ?

### DISCUSSION THREAD

**Discussion by brunoferromega** [kovidgoyal/kitty#7559] [Q&A]
I would like to use the layout divided(split) left oriented, but whenever I split a layout it changes to the right side first, when I exclude a window it return to right.
Is it possible configure that ? 
How can I do ?

**Comment by kovidgoyal** [comment]
Read the docs of the tall layout https://sw.kovidgoyal.net/kitty/layouts/#the-tall-layout

**Reply by brunoferromega** [reply]
Thank you...
I did read, but now that I got it!
